pom.xml

```diff
@@ -23,9 +23,9 @@
          xmlns="http://maven.apache.org/POM/4.0.0"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
-    <groupId>org.wso2.integration.connector</groupId>
-    <artifactId>mi-connector-googleads</artifactId>
-    <version>1.0.3-SNAPSHOT</version>
+    <groupId>org.wso2.carbon.esb.connector</groupId>
```

Comment:
Lets keep the groupId, artifactId as it is.

Reply:
Changed with https://github.com/wso2-extensions/esb-connector-googleads/pull/8/commits/b438836341d2f4b990fe90eda3ab36f906c9d957. and updated according to the https://github.com/wso2-extensions/esb-connector-googleads/commit/86485d1555575e6ecb6a990105393a1fcda4d869.